Formula/quack.rb

```diff
@@ -0,0 +1,25 @@
+require File.expand_path("../../Homebrew/emacs_formula", __FILE__)
+
+class Quack < EmacsFormula
+  desc "Enhanced Emacs support for Racket and Scheme"
+  homepage "http://www.neilvandyke.org/quack/"
+  url "http://www.neilvandyke.org/quack/quack.el"
```

Comment:
Unfortunately, I don't think there is one right now. Do I have to create a mirror on github?